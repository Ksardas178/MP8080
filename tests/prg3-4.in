LXI B b00001111 4 //загружаем 1030 в BC 00000100|00001111 
LXI D b11111101 3 //загружаем 3074 в DE 00000011|11111101
		//								00001000|00001100
XCHG //load value in DE to HL
DAD B //HL(DE) + BC -> to HL
SHLD o200 o000
HLT