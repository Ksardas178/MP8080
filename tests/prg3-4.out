
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 00000001
00000001: 00001111
00000010: 00000100
00000011: 00010001
00000100: 11111101
00000101: 00000011
00000110: 11101011
00000111: 00001001
00001000: 00100010
00001001: 10000000
00001010: 00000000
00001011: 01110110
